Content.Server/Silicons/Laws/SiliconLawSystem.cs

```diff
@@ -163,14 +163,14 @@ private void OnEmagLawsAdded(EntityUid uid, SiliconLawProviderComponent componen
         component.Lawset?.Laws.Insert(0, new SiliconLaw
         {
             LawString = Loc.GetString("law-emag-custom", ("name", name), ("title", Loc.GetString(component.Lawset.ObeysTo))), // DeltaV: pass name from variable
-            Order = -1 // Goobstation - AI/borg law changes - borgs obeying AI
+            Order = -2 // Goobstation - AI/borg law changes - borgs obeying AI
```

Comment:
Needs a DV comment for this, too. Append the change to the end and explain what's new